src/katgpucbf/fgpu/recv.py

```diff
@@ -255,6 +255,11 @@ def make_sensors(sensor_timeout: float) -> aiokatcp.SensorSet:
         for sensor in missing_sensors:
             TimeoutSensorStatusObserver(sensor, sensor_timeout, aiokatcp.Sensor.Status.NOMINAL)
             sensors.add(sensor)
+
+        sensors.add(
+            DeviceStatusSensor(sensors, "rx.device-status", "F-engine is receiving a good, clean digitiser stream")
+        )
```

Comment:
Should be outside the loop over `pol` - currently you're making two of these sensors (one of which will just be garbage-collected).

Reply:
Good catch!